modules/navigation/src/main/java/com/sygic/maps/module/navigation/listener/OnInfobarButtonsClickListener.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019 Sygic a.s. All rights reserved.
+ *
+ * This project is licensed under the MIT License.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.sygic.maps.module.navigation.listener
+
+import androidx.annotation.RestrictTo
+import androidx.lifecycle.LiveData
+import com.sygic.maps.module.navigation.infobar.InfobarButton
+
+/**
+ * Interface definition for a callback to be invoked when a click to the infobar button has been made.
+ */
+interface OnInfobarButtonsClickListener {
+
+    /**
+     * Modifies the infobar left button appearance. Icon and background drawable is required (icon and background tint can be
+     * optionally provided).
+     *
+     * @return your left [InfobarButton] to display or null if no left [InfobarButton] should be displayed.
+     */
+    fun getLeftButton(): InfobarButton? = null
```

Comment:
ja by som to nerobil ako 1 listener pre oba buttony. Co ked budu neskor 3? to tu budeme menit? Preco nema kazdy button svoj. Nemusis chciet mat ani 2 a budes tu musiet posielat `null` (ok je tam by default, ale aj tak)

Reply:
už je to zrefaktorováno ;)